test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -470,6 +470,53 @@ Describe "Additional tests for Import-Module with WinCompat" -Tag "Feature" {
             $LogPath | Should -FileContentMatch 'True'
         }
     }
+
+    Context "Tests around Windows PowerShell Compatibility module deny list" {
+        BeforeAll {
+            $pwsh = "$PSHOME/pwsh"
```

Comment:
This is not needed nor used

Reply:
It should be used.

Reply:
I recall a relatively recent PR that specifically made this change all around our test code; so I'll leave this as is.